### PR TITLE
Update consistency of language for custom fields

### PR DESCRIPTION
**Pull request by noon-dawg**
## Summary

This pull request updates the use of langauge related to custom fields.

## Checklist

- [X] Writing style
- [X] API design standards

### CODE REVIEW

**Review by giokaxo — DISMISSED** [review]

**Review by nanobash — DISMISSED** [review]

File: openapi/paths/custom-fields@{resource}.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ get:
   parameters:
     - $ref: ../components/parameters/collectionLimit.yaml
     - $ref: ../components/parameters/collectionOffset.yaml
-  description: Retrieves a schema of custom fields for the specified resource type.
+  description: Retrieves the custom fields schema for a specified resource schema.
```
Comment (ashkarpetin) [inline openapi/paths/custom-fields@{resource}.yaml:14]:
```suggestion
  description: Retrieves the custom fields schema for a specified resource.
```

File: openapi/paths/custom-fields@{resource}@{name}.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ get:
   summary: Retrieve a custom field
   operationId: GetCustomField
   x-sdk-operation-name: get
-  description: Retrieves a schema of a specified custom field for a specified resource type.
+  description: Retrieves a specified custom field for a specified resource schema.
```
Comment (ashkarpetin) [inline openapi/paths/custom-fields@{resource}@{name}.yaml:19]:
```suggestion
  description: Retrieves a specified custom field for a specified resource.
```

File: openapi/paths/custom-fields@{resource}@{name}.yaml [hunk 3]
```diff
@@ -45,13 +45,13 @@ put:
   summary: Create or alter a custom field
   operationId: PutCustomField
   x-sdk-operation-name: create
-  description: Creates or alters a schema of the specified custom field for the specified resource type.
+  description: Creates or alters a specified custom field for a specified resource schema.
```
Comment (ashkarpetin) [inline openapi/paths/custom-fields@{resource}@{name}.yaml:48]:
```suggestion
  description: Creates or alters a specified custom field schema for a specified resource.
```

File: openapi/paths/custom-fields@{resource}@{name}.yaml [hunk 4]
```diff
@@ -45,13 +45,13 @@ put:
...
   requestBody:
     content:
       application/json:
         schema:
           $ref: ../components/schemas/CustomField.yaml
-    description: Custom fields schema of the specified resource type.
+    description: Custom fields schema of the specified resource schema.
```
Comment (ashkarpetin) [inline openapi/paths/custom-fields@{resource}@{name}.yaml:54]:
```suggestion
    description: Custom fields schema of the specified resource.
```

File: openapi/paths/storefront/custom-fields@{resource}.yaml [hunk 5]
```diff
@@ -13,7 +13,7 @@ get:
   parameters:
     - $ref: ../../components/parameters/collectionLimit.yaml
     - $ref: ../../components/parameters/collectionOffset.yaml
-  description: Retrieves a schema of custom fields for the specified resource type.
+  description: Retrieves the custom fields schema for a specified resource schema.
```
Comment (ashkarpetin) [inline openapi/paths/storefront/custom-fields@{resource}.yaml:16]:
```suggestion
  description: Retrieves the custom fields schema for a specified resource.
```

**Comment by noon-dawg** [comment]
Thanks @ashkarpetin 🙏 
Applied all suggestions in [a6711ff](https://github.com/Rebilly/api-definitions/pull/1592/commits/a6711ff89bf18e89bc92567d76188c4e077bdf85)

**Review by ashkarpetin — APPROVED** [review]

**Review by justnero — APPROVED** [review]

**Pull request merged**